log/log.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2022 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"fmt"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+type Logger struct {
+	Zap *zap.Logger
```

Comment:
Maybe add a `// TODO` here to eventually stop exposing the `Zap` object and instead expose `Info()`, `Warn()`, `Sync()` methods directly.

Reply:
@azeemshaikh38 -- Would you prefer I add that in this PR or can I take it as a follow-up, post-merge?

Reply:
(nvm, tests failed, so I'll add it now 🙃)

Reply:
Whichever you prefer. You can simply leave a comment and address in a separate PR if you'd like.

Reply:
Added a comment in https://github.com/ossf/scorecard/pull/1502/commits/ac1d50f11a146f768fc1cb93777c061b40fb2d0b.